lib/capsize/deployer.rb

```diff
@@ -157,7 +179,7 @@ def write_stage
           f.puts "role :#{role.name}, %w{#{find_host_user(@project)}@#{role.host.name}}"
         end
         @stage.configuration_parameters.each do |parameter|
-          f.puts "set :#{parameter.name}, \"#{parameter.value}\""
+          f.puts print_parameter(parameter)
```

Comment:
These changes are nice — always better to have meaningful, self-documenting method calls than inline constants. Good job.